waiter/integration/waiter/metrics_output_test.clj

```diff
@@ -140,3 +140,71 @@
         (is (>= (get-in aggregate-metrics ["counters" "request-counts" "total"]) num-requests)))
 
       (delete-service waiter-url service-id))))
+
+
+(defmacro get-percentile-value
+  [metric p]
+  `(let [p# ~p
```

Comment:
Why create `p#` and not refer to `~p` everywhere?

Reply:
I think it's always good practice to do this in macros with arguments that are used more than once, to avoid re-evaluating the expression for `p`.

Reply:
Okay.